app/assets/components/contact/contact.js

```diff
@@ -0,0 +1,12 @@
+export default () => {
+  // Loops through dom and finds all elements with nhsuk-contact--clickable class
```

Comment:
I am not sure what this file is doing - I cant see a reference to `nhsuk-contact--clickable` anywhere else.

the comments here are a bit verbose - I think the code is clear enough without them.
Maybe a concise single comment describing the javascript if you think it needs explaining